docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -0,0 +1,208 @@
+---
+sidebar_title: BackgroundTask
+title: Background Task
+description: A library that provides an API for running background tasks.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-background-task'
+packageName: 'expo-background-task'
+platforms: ['android', 'ios']
+isNew: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { PlatformTags } from '~/ui/components/Tag/PlatformTags';
+
+`expo-background-task` provides an API to run deferrable background tasks in a way that optimizes battery and power consumption for the end user. This module uses the [`BGTaskScheduler`](https://developer.apple.com/documentation/backgroundtasks/bgtaskscheduler) API on iOS and the [`WorkManager`](https://developer.android.com/topic/libraries/architecture/workmanager) API on Android to schedule tasks. It also uses the [expo-task-manager](task-manager.mdx) Native API for running JavaScript tasks.
+
+## Background tasks
+
+A background task is a deferrable unit of work that is performed in the background, outside the lifecycle of your app. This is useful for tasks that need to be executed when the app is inactive, such as syncing data with a server, fetching new content, or even checking if there are any [Expo Updates](updates.mdx).
```

Comment:
```suggestion
A background task is a deferrable unit of work that is performed in the background, outside your app's lifecycle. This is useful for tasks that need to be executed when the app is inactive, such as syncing data with a server, fetching new content, or even checking if there are any [`expo-updates`](updates.mdx).
```

- Let's use the Expo package name here instead of the API. Generally, users are more familiar with the package names.
- We can be concise at the end of the first sentence.